Kickstarter-iOS/DataSources/ProjectPamphletContentDataSourceTests.swift

```diff
@@ -64,10 +65,22 @@ final class ProjectPamphletContentDataSourceTests: TestCase {
     }
   }
 
-  func testRewardsSection_nativeCheckoutFeature_showsWithTurnedOff() {
+  func testRewardsSection_showsWhen_nativeCheckoutFeatureEnabled_checkoutExperimentDisabled() {
```

Comment:
Should these tests cover the following combinations? (I think we might be missing one)

1. feature enable + experiment disabled
2. feature disabled + experiment enabled
3. feature disabled + experiment disabled (I think we're missing this one)